qatopics.md

```diff
@@ -0,0 +1,74 @@
+## Main Endpoint
+**GET /api/integration/qatopics**
+
+Provide access to the Quality Assurance Broker topics. It returns the list of the Quality Assurance Broker topics.
```

Comment:
Could we describe where these topics come from?  Do these topics get assigned/set by the qasource?  If so, we might say "It returns the list of Quality Assurance topics available from existing Quality Assurance Sources (see qasources endpoint)" or something similar.

I think it's very important in these docs to link up related endpoints so we describe *how* they are related.